test/sql/window/test_rank_orderby.test

```diff
@@ -0,0 +1,29 @@
+# name: test/sql/window/test_rank_orderby.test
+# description: Test argument ordering for RANK and PERCENT_RANK
+# group: [window]
+
+statement ok
+PRAGMA enable_verification
+
+query IIII
```

Comment:
Could we add a larger test? Large enough to trigger parallelism (could be with `PRAGMA verify_parallelism`).

Reply:
Heh. I'll try to think of one, but `RANK` is hard to check because it involves peers.